.cirrus.yml

```diff
@@ -225,6 +225,99 @@ task:
         gdb --batch --quiet -ex "thread apply all bt full" -ex "quit" $binary $corefile ;
       done
 
+task:
+  only_if: $CIRRUS_CHANGE_MESSAGE !=~ '.*\nci-os-only:.*' || $CIRRUS_CHANGE_MESSAGE =~ '.*\nci-os-only:[^\n]*linux.*'
+  container:
+    dockerfile: ci/docker/linux_opensuse_tumbleweed
+    cpu: 4
+    memory: 4G
+  env:
+    CCACHE_DIR: "/tmp/ccache_dir"
+    DEBUGINFOD_URLS: "https://debuginfod.opensuse.org/"
+
+  ccache_cache:
+    folder: "/tmp/ccache_dir"
+
+  sysinfo_script:
+    - id
+    - uname -a
+    - cat /proc/cmdline
+    - lsblk
+    - ulimit -a -H
+    - ulimit -a -S
+    - export
+  sysconfig_script:
+    - useradd -m -U postgres
+    - chown -R postgres:postgres .
+    - mkdir -p /tmp/ccache_dir
+    - chown -R postgres:postgres /tmp/ccache_dir
+    - echo '* - nofile 65536' > /etc/security/limits.d/postgres.conf
+    - su postgres -c 'ulimit -l -H'
+    - su postgres -c 'ulimit -l -S'
+    # - echo '/tmp/%e-%s-%p.core' > /proc/sys/kernel/core_pattern
```

Comment:
Does core dump handling actually work without this?

Reply:
It doesn't work. For it to work, docker needs to run on privileged mode and for docker to run on privileged mode, It should run on a VM. See: https://cirrus-ci.org/guide/docker-builder-vm/